### ISSUE TITLE
Unicorn adjustments

### ISSUE THREAD

**Issue by jasonLaster** [RecordReplay/devtools#6846]
**Description:**

* [ ] The circle badges should be 7px in the console
* [x] #6848: The unicorn should not be pushed down when paused

---

**Note** We should check with Jon and see if the badge was intentionally changed to 12px first

https://www.loom.com/share/ea7fb845139d4b0abd3643ab7537e300
https://www.figma.com/file/mAotNZRv6M5X0zHIFKPzFY/Replay-Design-Doc?node-id=12315%3A94607
https://www.loom.com/share/82d7bf7fcb674e5bbba835e521c00f48

**Comment by jasonLaster** [comment]
Thanks for the context, that's helpful!

> a) the special casing between shapes and colors was causing a lot of trouble

We can probably address this with a classname and css

> b) 7px was very small and made it hard to visually distinguish colors like yellow and orange

That's fair. Let's see how something in the middle feels. 9px or 10px. Print Statements are the crown jewl of Replay, I don't mind spending some cycles getting it just right.

**Comment by jonbell-lot23** [comment]
cc @jcmorrow FYI talking about this code structure again

**Comment by jonbell-lot23** [comment]
There are other things on my radar for making unicorn as great as possible:

* Selected states (see borders in original design ticket)
* Maybe some subtle animations for when you mouse over stuff
* A better palette icon
* A better dismiss icon -- right now it's just a plain white space you need to know to press

**Comment by jonbell-lot23** [comment]
Hey @jcmorrow @bvaughn and @jasonLaster, here's the design point of view:

* I am happy to move the circles to 9px rather than the 12px they are currently
* There are some other nice-to-haves that I'll formalise later (see above comment)
* I spent a full 3-4 days getting this just right. Colours accessible but still nice. Dark theme/light theme. Measuring the sizes to make sure the margins felt right. And a ton of refactoring. The PR is hilarious because there are so many dang commits as I worked to get everything just right. In the end, I asked @jcmorrow to help because the code is a bit hard to parse.
* All of that to say, I think the best use of resources is for someone else (not sure who) to make these changes. Last week was a good training exercise for me but I'm sure someone else can do it a lot faster :)

So that's where I am. Let's do 9px, and I think someone else is best to do it!

**Comment by bvaughn** [comment]
I think the only action item from the above comment is for me to change the circles from 12px to 9px (but leave the unicorn at 12px).

I don't see any open PRs from Jon regarding colors or anything. I believe you're talking about #6668 which is done so 👍🏼

**Comment by jasonLaster** [comment]
> Please don't touch (or resize) the unicorn icon! I put a lot of blood, sweat, and tears into getting that right!

This is getting mythical

**Comment by jonbell-lot23** [comment]
@bvaughn sorry for the wall of text here. Yup, all we needed was 9px but there was a lot of history I wanted to document :)

I am going to see if I can get better at tldr so you don't have to go digging so much.

@jasonLaster mythical is right! Did you know I built it in Figma by hand? This is ... not recommended 🤪

**Comment by bvaughn** [comment]
No worries @jonbell-lot23 😄 

Check out the frontend now that #6873 has landed. I tidied a lot of things up. If you want any follow ups though let's chat!

**Comment by jonbell-lot23** [comment]
Looks great @bvaughn , both the visuals and the code improvements. I did some small stuff, Josh pushed it further, but what you did is going to be so helpful moving forward. Thanks!

Aesthetic/subjective opinion: the original goal was to make the circles equal to the unicorn in weight. I think we've gone way in the other way, the unicorn looks MASSIVE compared to the circles. But that's cool with me! There are worse things than a proud unicorn in a console.

**Comment by jonbell-lot23** [comment]
Darn, I spoke too soon

![image](https://user-images.githubusercontent.com/9154902/169605830-759bf930-2a52-4ab4-913d-a835ffcbaa84.png)

The unicorn has a horn, and the pixel art requires crisp lines. Which means this is a visual bug:

![image](https://user-images.githubusercontent.com/9154902/169605979-d111b72f-92ee-4570-a4a0-0bc8d44ff31f.png)

Notice that the horn is being cut off. The way I handled this before is by removing the border-radius. But then the image needs to look right on the row, which can take some adjusting to get right, especially compared to the other rows dealing with different margins.

**Comment by bvaughn** [comment]
Ah, this is just a border-radius problem. My oversight. Apologies!
